hardware_interface/include/hardware_interface/handle.hpp

```diff
@@ -70,60 +78,58 @@ class ReadOnlyHandle
 
   double get_value() const
   {
+    // BEGIN (Handle export change): for backward compatibility
+    // TODO(Manuel) return value_ if old functionality is removed
     THROW_ON_NULLPTR(value_ptr_);
     return *value_ptr_;
+    // END
+  }
+
+  void set_value(double value)
+  {
+    // BEGIN (Handle export change): for backward compatibility
+    // TODO(Manuel) set value_ directly if old functionality is removed
+    THROW_ON_NULLPTR(this->value_ptr_);
+    *this->value_ptr_ = value;
+    // END
   }
 
 protected:
   std::string prefix_name_;
   std::string interface_name_;
+  HANDLE_DATATYPE value_;
```

Comment:
Just to confirm, in this version we define it here but not use it. Is this intended?